site/en/blog/how-fugu-is-my-browser-and-how-fugu-is-the-web/index.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: 'layouts/blog-post.njk'
+title: '"How Fugu is my browser?" and "How Fugu is the web?"'
+subhead: >
+  To answer these questions, check out a browser testing site and a browser extension, both focused
+  on the web's capabilities.
+date: 2023-01-16
+# updated: 2023-01-16
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/8FZcBmFowbDKWxpkOytx.jpg
+alt: Blowfish swarm swimming in the ocean.
+authors:
+  - thomassteiner
+tags:
+  - capabilities
+---
+
+## How Fugu is my browser?
+
+Many developers and executives alike don't realize how capable the modern web has become.
```

Comment:
I'd generally prefer not to open by telling people they don't know something. And the next sentence is a bit long and complicated. You can leave out the first sentence, and I'd suggest turning this around to be something like "Project Fugu APIs such as X and Y have brought capabilities to the web, that you might expect only to be possible in a phone app. For example, you can now <perhaps slightly simpler example here>"